tests/test_url_finders.py

```diff
@@ -0,0 +1,140 @@
+import pytest
+
+from freezeyt import freeze
+from freezeyt.freezer import parse_url_finders
+from freezeyt.url_finders import get_html_links
+from freezeyt.url_finders import get_css_links
+from testutil import context_for_test
+
+
+TEST_DATA = {
+    'html_links_fallback':
+        (   'text/html',
+            {'text/html': 'get_html_links'},
+            ('freezeyt.url_finders', 'get_html_links', get_html_links)
```

Comment:
```suggestion
            get_html_links,
```
No need to test `__module__` and `__name__` here.